### DISCUSSION TITLE
HELP!!! A question about "NoteName"

### DISCUSSION THREAD

**Discussion by Chen-JjinG** [naudio/NAudio#1115] [Q&A] [answered]
Why is my NoteName different from the one in DAW
![60eb2f66812cd3c38fcf32e8d771da2](https://github.com/naudio/NAudio/assets/114490297/1371edb6-9897-40ae-9f84-4788653e984a)

**Comment by Chen-JjinG** [comment]
Is there a discrepancies  MIDI note between Ableton Live and NAudio ?

**Comment by markheath** [comment, marked as answer]
there are two different conventions for middle C (MIDI note 60) - one is to call it C3, one is to call it C4, and different DAWs use different conventions. The DAW I used when I created the code that named the notes (over 20 years ago!) was Cakewalk. No idea what that used. In any case, this looks like a bug that no one has spotted in a long time! The code calculating the octave number is just doing: `int octave = noteNumber / 12;`

**Reply by Chen-JjinG** [reply]
thank you sir,will this be fixed in the next version?

**Reply by markheath** [reply]
Maybe, although it's very easy to write your own code to name a note from its MIDI number the way you want to
